Examples/Movies/SearchScreen.js

```diff
@@ -13,34 +13,32 @@
  *
  * @flow
  */
-'use strict';
 
-var React = require('react-native');
-var {
+import React, {
   ActivityIndicatorIOS,
+  Component,
   ListView,
   Platform,
   ProgressBarAndroid,
   StyleSheet,
   Text,
   View,
-} = React;
-var TimerMixin = require('react-timer-mixin');
+} from 'react-native';
 
-var invariant = require('invariant');
-var dismissKeyboard = require('dismissKeyboard');
+import invariant from 'invariant';
+import dismissKeyboard from 'dismissKeyboard';
 
-var MovieCell = require('./MovieCell');
-var MovieScreen = require('./MovieScreen');
-var SearchBar = require('SearchBar');
+import MovieCell from './MovieCell';
+import MovieScreen from './MovieScreen';
+import SearchBar from './SearchBar';
```

Comment:
Ah yeah, I think Flow doesn't recognize the `.ios.js` and `.android.js` extensions unfortunately, which SearchBar is implemented with. I guess let's move back to the `import SearchBar from 'SearchBar'` syntax for now until there's a better solution to support those variants in flow.